docs/user-guide/permissions/index.md

```diff
@@ -0,0 +1,8 @@
+Atomist integrates with and automates activities over various tools and services in your development toolchain. To do so, certain permission are needed. We aim to ask only for the permissions required to get the job done.
```

Comment:
Suggestion for the last sentence: "Atomist only asks for permissions required to accomplish its work."